docs/sections/community/contributing.rst

```diff
@@ -30,21 +30,28 @@ Environment Setup
 
     $ brew install yarn
 
-4. Clone the dagster repository to the destination of your choice:
+4. Ensure that you have pandoc installed by running `pandoc -v`.  If you are on macOS, you can install pandoc with Homebrew:
```

Comment:
Could we also add a link to https://pandoc.org/installing.html in case they're on a different platform?